python/pyarrow/parquet/core.py

```diff
@@ -1038,9 +1038,9 @@ def write(self, table_or_batch, row_group_size=None):
         ----------
         table_or_batch : {RecordBatch, Table}
         row_group_size : int, default None
-            Maximum size of each written row group. If None, the
-            row group size will be the minimum of the input
-            table or batch length and 64 * 1024 * 1024.
+            Maximum # of rows in each written row group. If None,
```

Comment:
```suggestion
            Maximum number of rows in each written row group. If None,
```

(I suppose "#" is quite universal, but just using the word "number" seems even more explicit and not much longer

Reply:
I agree.  I've switched to `number`.